lib/spack/docs/known_issues.rst

```diff
@@ -0,0 +1,16 @@
+============
+Known issues
+============
+
+Below is the list of known issues:
+
+#. Current conretization algorithm does not take into account default values
+   of variants when adding extra constrains to the spec via CLI. For example
+   'spack spec dealii@8.3.0 ^trilinos@12.4.2' will complain that 'deal.II'
+   does not depend on 'trilinos', although the package contains
+   '''
+   depends_on("trilinos",         when='+trilinos+mpi~int64')
+   depends_on("trilinos~hypre",   when="+trilinos+mpi+int64")
+   '''
+   A workaround is to make the spec more specific:
```

Comment:
i know that, by 

> more specific

I meant that you need to restrict the free of conretizer by, exactly, activating those variants.